pkg/codegen/testing/test/testdata/docs-collision/schema.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
Should add a conflicting module to this as well to check that also works as expected

Reply:
look good?